LICENSE.md.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
Same as above - clarifying if we generally want to include .meta files?

Reply:
Yes, it is recommended by unity https://docs.unity3d.com/Manual/ExternalVersionControlSystemSupport.html